### PR TITLE
Add laravel ui, layouts default and layouts home

### PR DESCRIPTION
**Pull request by nchanh**
Add laravel ui, layouts default and layouts home

### CODE REVIEW

File: src/public/js/gmaps.js [hunk 1]
```diff
@@ -0,0 +1,2065 @@
+(function(root, factory) {
+  if(typeof exports === 'object') {
+    module.exports = factory();
+  }
+  else if(typeof define === 'function' && define.amd) {
```
Comment (chien-rikai) [inline src/public/js/gmaps.js:5]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/rikai-academy/Foods_Drinks_01/pullRequest?prid=7829589)
```suggestion
  else if(typeof define === "function" && define.amd) {
```

File: src/public/js/price-range.js [hunk 2]
```diff
@@ -0,0 +1,388 @@
...
+		}
+
+		var tooltip = this.element.data('slider-tooltip')||options.tooltip;
+
+		this.tooltip = this.picker.find('.tooltip');
+		this.tooltipInner = this.tooltip.find('div.tooltip-inner');
+
+		this.orientation = this.element.data('slider-orientation')||options.orientation;
+		switch(this.orientation) {
+			case 'vertical':
```
Comment (chien-rikai) [inline src/public/js/price-range.js:50]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/rikai-academy/Foods_Drinks_01/pullRequest?prid=7829589)
```suggestion
			case "vertical":
```

File: src/public/js/contact.js [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+    });
+
+    var image = 'images/map-icon.png';
+    map.addMarker({
+        lat: 43.1580159,
+        lng: -77.6030777,
+        //icon: image,
+        animation: google.maps.Animation.DROP,
+        verticalAlign: 'bottom',
+        horizontalAlign: 'center',
```
Comment (chien-rikai) [inline src/public/js/contact.js:43]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/rikai-academy/Foods_Drinks_01/pullRequest?prid=7829589)
```suggestion
        horizontalAlign: "center",
```

File: src/resources/lang/vi/custom.php [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+    'logout' => 'Đăng xuất',
+    'logged_in_as' => 'Đăng nhập bởi',
+    'manager' => 'Quản lý',
+    'user' => 'Người dùng',
+    'users' => 'Người dùng',
+    'category' => 'Danh mục',
+    'product' => 'Sản phẩm',
+    'products' => 'Sản phẩm',
+    'order' => 'Đơn hàng',
+    'orders' => 'Đơn hàng',
```
Comment (namtprikai) [inline src/resources/lang/vi/custom.php:22]: Chung 1 nghĩa thì tạo ra 1 thôi

File: src/public/js/web/html5shiv.js [hunk 5]
```diff
@@ -0,0 +1,121 @@
...
+        shivCSS: !1 !== k.shivCSS,
+        supportsUnknownElements: g,
+        shivMethods: !1 !== k.shivMethods,
+        type: "default",
+        shivDocument: q,
+        createElement: p,
+        createDocumentFragment: function (a, b) {
+            a || (a = f);
+            if (g) return a.createDocumentFragment();
+            for (
```
Comment (chien-rikai) [inline src/public/js/web/html5shiv.js:106]: Codacy has a fix for the issue: [Expected { after 'for' condition.](https://app.codacy.com/gh/rikai-academy/Foods_Drinks_01/pullRequest?prid=7829589)
```suggestion
            for (
```

File: src/public/js/web/gmaps.js [hunk 6]
```diff
@@ -0,0 +1,2343 @@
...
+            } else {
+                for (var i = 0, latlng; (latlng = points[i]); i++) {
+                    path.push(new google.maps.LatLng(latlng[0], latlng[1]));
+                }
+            }
+        }
+
+        var polyline_options = {
+            map: this.map,
+            path: path,
```
Comment (chien-rikai) [inline src/public/js/web/gmaps.js:999]: Codacy has a fix for the issue: [Expected property shorthand.](https://app.codacy.com/gh/rikai-academy/Foods_Drinks_01/pullRequest?prid=7829589)
```suggestion
            path,
```

File: src/public/js/web/gmaps.js [hunk 7]
```diff
@@ -0,0 +1,2343 @@
...
+            var buildContextMenuHTML = function (control, e) {
+                var html = "",
+                    options = window.context_menu[self.el.id][control];
+
+                for (var i in options) {
+                    if (options.hasOwnProperty(i)) {
+                        var option = options[i];
+
+                        html +=
+                            '<li><a id="' +
```
Comment (chien-rikai) [inline src/public/js/web/gmaps.js:293]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/rikai-academy/Foods_Drinks_01/pullRequest?prid=7829589)
```suggestion
                            "<li><a id=\"" +
```

File: src/public/js/web/price-range.js [hunk 8]
```diff
@@ -0,0 +1,445 @@
...
+ * limitations under the License.
+ * ========================================================= */
+
+!(function ($) {
+    var Slider = function (element, options) {
+        this.element = $(element);
+        this.picker = $(
+            '<div class="slider">' +
+                '<div class="slider-track">' +
+                '<div class="slider-selection"></div>' +
```
Comment (chien-rikai) [inline src/public/js/web/price-range.js:26]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/rikai-academy/Foods_Drinks_01/pullRequest?prid=7829589)
```suggestion
                "<div class=\"slider-selection\"></div>" +
```

Comment (chien-rikai) [inline src/public/js/web/price-range.js:26]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/rikai-academy/Foods_Drinks_01/pullRequest?prid=7829589)
```suggestion
                "<div class=\"slider-selection\"></div>" +
```

**Review by namtprikai — APPROVED** [review]

**Review by namtprikai — APPROVED** [review]

**Pull request merged**